CHANGELOG.md

```diff
@@ -42,6 +42,7 @@
 - Fixes bug where Migrator \$execute() always appends semi-colon - [#924](https://github.com/cfwheels/cfwheels/issues/#924) [Adam Chapman]
 - Fixes bug where model createdAt property is changed upon update - [#927](https://github.com/cfwheels/cfwheels/issues/#927) [Brandon Shea, Adam Chapman]
 - Fixed silent application.wheels scope exception hampering autoMigrateDatabase - [#957](https://github.com/cfwheels/cfwheels/issues/#957) [Adam Chapman, Tom King]
+- Fixed pagination order ambiguous column name exception - [#957](https://github.com/cfwheels/cfwheels/issues/#980) [Adam Chapman, Mike Lange]
```

Comment:
```suggestion
- Fixed pagination order ambiguous column name exception - [#980](https://github.com/cfwheels/cfwheels/issues/#980) [Adam Chapman, Mike Lange]
```

Reply:
Since this didn't make it into the beta, can you start a new 2.1 section in the changelog?

Reply:
Can do!

Reply:
@neokoenig Is e1a4974 correct?

Reply:
Hey @chapmandu, do we need to modify or add a test for this?

Reply:
Yeah, I think we need a test to catch regressions.

Reply:
Test case added

Reply:
Looks like SQL Server doesn't like the CASE statement in the order by clause